### ISSUE TITLE
About parameter trainable for classifier

### ISSUE THREAD

**Issue by mathxin** [CyberZHG/keras-bert#53]
Hi. When using Bert model to classify tasks, the following code

- If `trainable = True` is set and then any classifier is added, the result will only be biased towards a certain category, whether multi-classification or bi-classification. 
`Non-trainable params: 0` in the result of `model. summary ()`.

- But if `trainable` is not set, the other codes are exactly the same, and the result is quite normal. At this point `Non-trainable params: 101, 306, 880`

So how to train fine-tune correctly?

	bert_model = load_trained_model_from_checkpoint(config_path, checkpoint_path, seq_len=maxlen, 
	                              training=False, trainable=True)
	pool_layer = MaskedGlobalMaxPool1D(name='Pooling')(bert_model.output)
	out = Dense(32, activation='relu')(pool_layer)
	output = Dense(units=class_num, activation=output_activation)(out)
	model = Model(bert_model.input, output)
	model.compile(loss=model_loss, optimizer=model_optimizer, metrics=['categorical_accuracy'])

**Comment by CyberZHG** [comment]
Generally, you don't need to tune the trained part of the model (because it is too large for your task).

**Comment by keapoa** [comment]
if I want to tune,so how I do?

**Comment by CyberZHG** [comment]
@keapoa 
See the following demo:

https://colab.research.google.com/github/CyberZHG/keras-bert/blob/master/demo/load_model/keras_bert_load_and_fit_tpu.ipynb
